addons/huntir/CfgWeapons.hpp

```diff
@@ -0,0 +1,35 @@
+
+class CfgWeapons {
+    class ACE_ItemCore;
+    class InventoryItem_Base_F;
+    
+    class ACE_HuntIR_monitor: ACE_ItemCore {
+        scope = 2;
+        displayName = $STR_ACE_HUNTIR_MON;
+        picture = PATHTOF(UI\w_huntir_monitor_ca.paa);
+        descriptionShort = $STR_ACE_HUNTIR_MON;
+        model = PATHTOF(data\ace_huntir_monitor.p3d);
+
+        class ItemInfo: InventoryItem_Base_F {
+            mass = 20;
+        };
+    };
+    
+    class GrenadeLauncher;
+    class UGL_F: GrenadeLauncher {
+        //magazines[] += {"ACE_HuntIR_M203"};
+        magazines[] = {"1Rnd_HE_Grenade_shell","UGL_FlareWhite_F","UGL_FlareGreen_F","UGL_FlareRed_F","UGL_FlareYellow_F","UGL_FlareCIR_F","1Rnd_Smoke_Grenade_shell","1Rnd_SmokeRed_Grenade_shell","1Rnd_SmokeGreen_Grenade_shell","1Rnd_SmokeYellow_Grenade_shell","1Rnd_SmokePurple_Grenade_shell","1Rnd_SmokeBlue_Grenade_shell","1Rnd_SmokeOrange_Grenade_shell","ACE_HuntIR_M203"};
```

Comment:
Does `+=` not work properly here? Are the magazines defined in `GrenadeLauncher`?